src/inc/til/small_vector.h

```diff
@@ -880,8 +896,8 @@ namespace til
         }
 
         T* _data;
-        size_t _capacity = N;
-        size_t _size = 0;
+        size_t _capacity;
+        size_t _size;

```

Comment:
By not initializing these fields, the move constructor avoids writing to these members twice.